app/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "app",
+  "version": "1.0.0",
+  "main": "src/index.js",
+  "license": "MIT",
+  "dependencies": {
+    "@aragon/api": "^1.0.0-beta.1",
+    "@aragon/api-react": "^1.0.0-beta.1",
+    "@aragon/ui": "^0.32.0",
+    "@babel/polyfill": "^7.2.5",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "rxjs": "^6.2.1",
+    "styled-components": "^4.1.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.3.4",
+    "@babel/plugin-proposal-class-properties": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "parcel-bundler": "^1.12.0"
+  },
+  "scripts": {
+    "build": "npm run sync-assets && npm run build:app && npm run build:script",
+    "build:app": "parcel build index.html -d ../dist/ --public-url \".\" --no-cache",
+    "build:script": "parcel build src/script.js -d ../dist/ --no-cache",
+    "devserver": "parcel serve index.html -p 8001 --out-dir ../dist/ --no-cache",
+    "start": "npm run sync-assets && npm run build:script -- --no-minify && npm run devserver",
+    "sync-assets": "copy-aragon-ui-assets ../dist"
```

Comment:
I'm wondering if building it to the root directory will be more confusing for people, since they're probably used to seeing the build being in the same folder as their `src`.

What's the default build folder for the CLI @0x6431346e? Is it `build/` in the root folder?

Reply:
> What's the default build folder for the CLI @0x6431346e? Is it build/ in the root folder?

If you mean where is the compiled CLI code, that's to `./dist`.
When doing `aragon run`/`publish`, etc. we don't have a specific directory, we upload the root and the GUI looks in `manifest.json` for `start_url` and `script`.

> I'm wondering if building it to the root directory will be more confusing for people, since they're probably used to seeing the build being in the same folder as their src.

Right...I can see `aragon-apps` has changed from `./dist` to `./app/build` for the compiled frontends. 🤔 
For me, it took some time to get used to having the "backend" in the same directory as the "frontend". I think "app" is also not the best name as we use it to describe the contracts as well. e.g.: "I have installed the Vault app".

Would it make sense for this to be a monorepo? I don't like the added complexity - which can overwhelm someone who is just getting started, but I feel like it would look very clean:

```
┬ root 
├─┬ packages 
│ ├─┬ webapp 
│ │ ├── build
│ │ ├── src
│ │ └── package.json 
│ ├─┬ contracts 
│ │ ├── build
│ │ ├── src
│ │ └── package.json 
│ ├─┬ templates 
│ │ └── ... 
│ ├─┬ cli 
│ │ └── ... 
│ ├─┬ api 
│ │ └── ... 
├── package.json 
├── README.md 
├── arapp.json
└── manifest.json
```

This way we can also include a CLI extension example, e.g: `aragon counter increment` 😁.

I will open a new issue to take the conversation there.

Reply:
We had a discussion around the monorepo structure in https://github.com/aragon/aragon-app-dx/issues/1, but I'm starting to think that this structure really starts to complicate things if you want to use `aragon apm publish` (as it is right now, since it needs to compile the contracts and create their artifact.json, etc).

Reply:
Oh...We need to improve `aragon apm publish` then, so it doesn't restrict certain folder structures...

Reply:
It isn't clear for me what `aragon apm publish` when there are both `package.json` and `app/package.json`. Does `aragon apm publish` publish in this case two apps? or whatever?